### PR TITLE
Add support for vlanconfd and intfconfd

### PR DESCRIPTION
**Pull request by JipanYanga**
Signed-off-by: Jipan Yang <jipan.yang@alibaba-inc.com>

**- What I did**
VLAN trunk support.

**- How I did it**

**- How to verify it**
VLAN trunk test plan: https://github.com/Azure/SONiC/wiki/VLAN-trunk-test-plan
**- Description for the changelog**

### CODE REVIEW

**Comment by JipanYanga** [comment]
Has dependency on:
https://github.com/Azure/sonic-swss/pull/356 
https://github.com/Azure/sonic-swss/pull/357

**Comment by JipanYanga** [comment]
image

**Comment by lguohan** [comment]
please resolve conflict.

File: dockers/docker-orchagent/base_image_files/cfgmgr [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker exec -i swss cfgmgr "$@"
```
Comment (lguohan) [inline dockers/docker-orchagent/base_image_files/cfgmgr]: it is better to incorporate this into the show/config commands in sonic utilities. I suggest to remove this in this PR and only include vlanmgrd and intfmgrd.

File: platform/broadcom/docker-orchagent-brcm.mk [hunk 2]
```diff
@@ -16,3 +16,4 @@ $(DOCKER_ORCHAGENT_BRCM)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
 $(DOCKER_ORCHAGENT_BRCM)_RUN_OPT += -v /var/log/swss:/var/log/swss:rw
 
 $(DOCKER_ORCHAGENT_BRCM)_BASE_IMAGE_FILES += swssloglevel:/usr/bin/swssloglevel
+$(DOCKER_ORCHAGENT_BRCM)_BASE_IMAGE_FILES += cfgmgr:/usr/bin/cfgmgr
```
Comment (lguohan) [inline platform/broadcom/docker-orchagent-brcm.mk]: remove

File: platform/cavium/docker-orchagent-cavm.mk [hunk 3]
```diff
@@ -16,3 +16,4 @@ $(DOCKER_ORCHAGENT_CAVM)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
 $(DOCKER_ORCHAGENT_CAVM)_RUN_OPT += -v /var/log/swss:/var/log/swss:rw
 
 $(DOCKER_ORCHAGENT_CAVM)_BASE_IMAGE_FILES += swssloglevel:/usr/bin/swssloglevel
+$(DOCKER_ORCHAGENT_BRCM)_BASE_IMAGE_FILES += cfgmgr:/usr/bin/cfgmgr
```
Comment (lguohan) [inline platform/cavium/docker-orchagent-cavm.mk]: remove

File: dockers/docker-orchagent/base_image_files/cfgmgr (hunk 1 above)
Comment (JipanYanga) [reply dockers/docker-orchagent/base_image_files/cfgmgr]: removed.

File: platform/broadcom/docker-orchagent-brcm.mk (hunk 2 above)
Comment (JipanYanga) [reply platform/broadcom/docker-orchagent-brcm.mk]: done.

File: platform/cavium/docker-orchagent-cavm.mk (hunk 3 above)
Comment (JipanYanga) [reply platform/cavium/docker-orchagent-cavm.mk]: done.

**Comment by lguohan** [comment]
can you also update the submodule in this PR?

**Comment by JipanYanga** [comment]
Updated with swss and swss-common submodules.

**Pull request merged**